docs/resource-schemas.md

```diff
@@ -321,6 +332,37 @@ have the following values:
 | `cross-platform` | Runs natively on iOS, Android, and/or desktop. |
 | `web`            | Runs on the web.                               |
 
+#### Asset docs
+
+Asset docs is used to determine which supporting documentation is available to fetch and render as
+page tabs in the asset details for the asset as well as it's location. For each optional document
+key, a path should be included; The path **must** be either a Github url according to the
```

Comment:
Can you find-all replace `Github url` with `GitHub URL` from proper capitalization?

Reply:
The schema.json files too.

Reply:
done!